src/node.cc

```diff
@@ -2843,6 +2844,11 @@ void SetupProcessObject(Environment* env,
     READONLY_PROPERTY(process, "_print_eval", True(env->isolate()));
   }
 
+  // -c, --check
+  if (syntax_check_only) {
+    READONLY_PROPERTY(process, "_syntax_check_only", True(env->isolate()));
```

Comment:
Oh okay then. I was just worried that we would introduce non camel case identifiers in to the js land.